src/solve.jl

```diff
@@ -92,32 +92,29 @@ function DiffEqBase.__init(
                             mass_matrix = prob.f.mass_matrix)
     end
 
-    # absolut tolerance for fixed-point iterations has to be of same type as elements of u
-    # in particular important for calculations with units
-    if typeof(alg.fixedpoint_abstol) <: Nothing
-        fixedpoint_abstol_internal = map(eltype(uType), integrator.opts.abstol)
+    # define absolut tolerance for fixed-point iterations
+    if alg.fixedpoint_abstol === nothing
+        fixedpoint_abstol_internal = integrator.opts.abstol
     else
```

Comment:
I guess we should copy abstol.

Reply:
why copy?

Reply:
I was thinking about use cases were you want to adjust the error tolerances of the integrator adaptively and I guess it's not (always) intended that the tolerances of the fixed point iterations change in the same way.